requirements-dev.txt

```diff
@@ -1,4 +1,5 @@
+# include requirements.txt so pip has context to avoid installing incompatible dependencies
+-r requirements.txt
 pytest
 transformers     >= 4.9.2,<=4.24.0
 tensorflow_text  >=2.5.0
-protobuf         >=3.12.2,<=3.20.1
```

Comment:
this restriction is introduced by some version of ONNX, but I am not sure if it is fixed in ONNX now.

Reply:
I think later versions of ONNX specify the protobuf dependency more correctly